util/grpc/errors_test.go

```diff
@@ -109,11 +109,10 @@ func Test_kubeErrToGRPC(t *testing.T) {
 				return apierr.NewServerTimeout(schema.GroupResource{}, "update", 1)
 			},
 			expectedErrFn: func() error {
-				err := apierr.NewServerTimeout(schema.GroupResource{}, "update", 1)
-				grpcStatus := status.New(codes.Unavailable, err.Error())
+				grpcStatus := status.New(codes.Unavailable, "Command timed out before the specified conditions were met")
```

Comment:
Ah I think I see the issue, the error should still be the same as before. So 

```suggestion
				err := apierr.NewServerTimeout(schema.GroupResource{}, "update", 1)
				grpcStatus := status.New(codes.Unavailable, err.Error())
```

Reply:
Oh, I see. Because it will still get the error directly from the `kubeErrToGRPC` function, am I right?

Reply:
Actually now that I've looked at it more closely, you need a new test checking for `apierr.IsTimeout` (since `apierr.IsTimeout` != `apierr.IsServerTimeout`).